WPFSKillTree/SkillTreeFiles/AscendancyClasses.cs

```diff
@@ -3,14 +3,9 @@
 
 namespace POESKillTree.SkillTreeFiles
 {
-    public class AscendancyClasses
+    public static class AscendancyClasses
```

Comment:
It's more about dependencies from other classes on these. If they are passed as instances (e.g. via `ISkillTree`), they can just be mocked in tests. When they are used statically, there's no way but to deal with them.

If you can't change it easily, you don't have to. It's not worse than before, it would just be a good way to make it better, imo.